src/components/js/Homepage.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import Form from './Form';
+import '../css/Homepage.css';
+import Todos from './Todos';
+import NavLinks from './NavLinks';
+
+export default function Home() {
+  const [todo, setTodo] = useState('');
+  const [storage, setStorage] = useState(JSON.parse(localStorage.getItem('todos')) ? JSON.parse(localStorage.getItem('todos')) : []);
+  const handleChange = (event) => {
+    setTodo(event.target.value);
+  };
+
+  const handleRemove = (event) => {
+    const { id } = event.target.parentNode.parentNode.parentNode.parentNode;
```

Comment:
- [ ] I think this train of `parentNode` 😆. When you wanna call the `handleRemove` function you pass the id since you have access to the to-do object.
```javascript
 const handleRemove = (id) => {
```

Reply:
That was a good one 😆😆😆😆. I have implemented the change. 👍  One more thing, I am using FontAwesome library so I need to use parentNode. So, I am going to use it once.